smarty/sampler.py

```diff
@@ -129,41 +144,16 @@ def __init__(self, molecules, basetypes_filename, initialtypes_filename, decorat
                 self.atomtypes = [(smarts, typename)] + self.atomtypes
```

Comment:
@camizanette - yes, I agree with Caitlin on this change. These are two separate issues. But the code change she just recommended is needed, otherwise the order of added base types may get jumbled upon addition, and they should be preserved and put at the top of the list. (The initial types would be more specialized, typically, whereas base types could be elemental/generic.)